Cubical/Structures/Algebra.agda

```diff
@@ -0,0 +1,231 @@
+{-# OPTIONS --cubical --safe #-}
```

Comment:
Please add `--no-import-sorts` here and to the OPTIONS for the other files in this PR